src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -130,6 +129,9 @@ private async Task CreateBlocksAsync()
             {
                 try
                 {
+                    this.logger.LogTrace("IsInitialBlockDownload={0}, AnyConnectedPeers={1}, BootstrappingMode={2}, IsFederationMember={3}",
```

Comment:
logging style is violated, there is a race condition (IsIBD might be logged true but in the if statement it might be false, so better to save value first to a variable)